canvas_modules/harness/cypress/support/canvas/object-model-cmds.js

```diff
@@ -90,3 +140,10 @@ Cypress.Commands.add("clearMessagesFromAllNodes", () => {
 			});
 	});
 });
+
+Cypress.Commands.add("isObjectModelEmpty", () => {
```

Comment:
We'd usually expect a command beginning with `is` to return a boolean whereas this is returning a count so I think the name should be `getObjectCountFromObjectModel`